appinventor/appengine/src/com/google/appinventor/client/editor/simple/palette/ComponentRemoveWidget.java

```diff
@@ -21,9 +21,16 @@ public class ComponentRemoveWidget extends AbstractPaletteItemWidget {
   private static final ImageResource imageResource = Ode.getImageBundle().deleteComponent();
 
   private static final Ode ode = Ode.getInstance();
+  String name;
```

Comment:
See discussion on the last two comments about whether this is actually useful in practice.